maven-model-builder/pom.xml

```diff
@@ -87,6 +87,11 @@ under the License.
       <artifactId>xmlunit-matchers</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
:pray: why mocking there? this is a hell anywhere in maven code base and ultimately the related tests are useless in general so let's try to avoid it